e2e-tests/portal/cypress/e2e/journeys/maker/proceed-to-review/proceed-to-review.spec.js

```diff
@@ -161,25 +124,21 @@ context('Ensure proceed to review button is only visible once facilities are in
     cy.clickSubmitButton();
 
     // Navigate to the deal in question
-    pages.contract.visit(deal);
+    dashboardDeals.visit();
+    cy.clickDashboardDealLink();
 
     // Ensure facility stage is `Incomplete`
-    pages.contract.loansTransactionsTableRows().each((row, index) => {
+    contract.loansTransactionsTableRows().each((row) => {
       const loanId = row.attr('data-cy').split('-')[1];
-      const loan = pages.contract.loansTransactionsTable.row(loanId);
-      const status = index === 2 ? 'Incomplete' : 'Complete';
-      loan.loanStatus().contains(status);
+      const loan = contract.loansTransactionsTable.row(loanId);
+      loan.loanStatus().contains('Incomplete');
     });
```

Comment:
This seems a bit odd, are there no existing selectors we can use for this logic?

Reply:
We are already using existing selectors from e2e-tests\portal\cypress\e2e\pages\contract.js